.changes/next-release/Feature-cf59acbb-0c3a-4299-9996-c9f83326353e.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Toolkit dynamically chooses an open port when debugging SAM applications"
```

Comment:
Consider adding to the end ... "instead of always using port 5858"
Also: replace "open" with "available" or "unused"